docs/designers-developers/developers/tutorials/metabox/meta-block-4-use-data.md

```diff
@@ -0,0 +1,31 @@
+# Use Post Meta Data
+
+You can use the post meta data stored in the last step in multiple ways.
+
+## Use Post Meta in PHP
+
+The first example uses the value from the post meta field and appends it to the end of the post content wrapped in a H4 tag. This method in PHP is unchanged in WordPress 5.0.
+
+```php
+function myguten_content_filter( $content ) {
+	$value = get_post_meta( get_the_ID(), 'myguten_meta_block_field', true );
+	return sprintf( "%s <h4> %s </h4>", $content, esc_html( $value ) );
+}
+add_filter( 'the_content', 'myguten_content_filter' );
+```
+
+## Use Post Meta in Block
+
+You can also use the post meta data in other blocks. For this example the data is loaded at the end of every paragraph block when it is rendered, ie. shown to the user. You can replace this for any core or custom block types as needed.
+
+```php
+function myguten_render_paragraph( $attributes, $content ) {
+	$value = get_post_meta( get_the_ID(), 'myguten_meta_block_field', true );
+	return sprintf( "%s (%s)", $content, esc_html( $value ) );
```

Comment:
In going through both examples, there was a small mismatch between how the code works and the explanation. I'd like to stress here somewhere that if the post meta doesn't exist for a particular post, `myguten_meta_block_field` will be void. Or perhaps changing the example to not print anything if the post doesn't have a registered `myguten_meta_block_field`?

Reply:
:+1: Updated in 581ff5395